updater.c

```diff
@@ -1420,15 +1420,14 @@ int pv_update_finish(struct pantavisor *pv)
 	switch (u->status) {
 	// DONE TRANSITIONS
 	case UPDATE_TESTING_REBOOT:
-		if (pv_bootloader_set_commited(pv->state->rev)) {
+		if (pv_bootloader_commit_update(pv->state->rev)) {
 			pv_log(ERROR,
-			       "revision for next boot could not be set");
+			       "revision could not be committed to bootloader");
 			ret = -1;
 			goto out;
 		}
 		pv_update_set_status(u, UPDATE_DONE);
 		pv_storage_set_rev_done(pv->state->rev);
-		pv_bootloader_commit_update();
```

Comment:
@anibalportero lets not remove the two teps for committing. i had thought a while about this for rpiab case and did it so we are safe in case reboot happen between the set_committed and the set_rev